packages/syncapi/src/customSyncStorage.ts

```diff
@@ -0,0 +1,34 @@
+import { HasId, Store } from '@raincatcher/store';
```

Comment:
This was planned to be included for cases when custom storage is required. 
I'm not sure if we need any cloud init support at the moment.